plugin/inlay_hint.py

```diff
@@ -97,7 +97,8 @@ def inlay_hint_to_phantom(view: sublime.View, inlay_hint: InlayHint, session: Se
 
 
 def get_inlay_hint_html(view: sublime.View, inlay_hint: InlayHint, session: Session, phantom_uuid: str) -> str:
-    label = format_inlay_hint_label(inlay_hint, session, phantom_uuid)
+    truncate_limit = view.settings().get("inlay_hints_truncate_limit", userprefs().inlay_hints_truncate_limit)
```

Comment:
What is the purpose to try to read this setting from the view settings first - so that you could set different values per syntax? This is not really something which is done this way with any of the other LSP settings I think, and I don't imagine it to be very useful.

Reply:
Done in [1ba72a6](https://github.com/sublimelsp/LSP/pull/2514/commits/1ba72a66d41777c5b0f4ad237b149efba63b16ad)